foundry.toml

```diff
@@ -5,15 +5,14 @@ test = "test"
 solc = "0.8.23"
 evm_version = "paris"
 
-libs = [ "./lib" ]
+libs = ["./lib"]
 
 [profile.ir]
 via_ir = true
 optimizer = true
 optimizer_runs = 100000000
+bytecode_hash = "none"
+cbor_metadata = false
 
 [fmt]
 ignore = ["src/vendor/**/*"]
-
-bytecode_hash = "none"
-cbor_metadata = false
```

Comment:
Good catch. I think we should move these two to the `default` profile, since it's not specific to `ir` and means it will be inherited by all other profiles.